crates/cli/commands/Cargo.toml

```diff
@@ -88,7 +88,8 @@ reth-discv4.workspace = true
 
 [features]
 default = []
-dev = [
+dev = ["arbitrary"]
+arbitrary = [
```

Comment:
this could have something to do with the name `dev`

we should just rename `dev` to arbitrary because the dev feature here only enables arbitrary related features